packages/replay-web/src/draw.ts

```diff
@@ -254,16 +250,14 @@ const drawUtils = (ctx: CanvasRenderingContext2D) => ({
       ctx.stroke();
     }
   },
-  text(
-    font: TextureFont,
-    text: string,
-    align: "left" | "center" | "right",
-    fillStyle: string
-  ) {
-    const fontString = `${font.size}px ${font.name}`;
+  text(font: TextureFont, text: string, fillStyle: string) {
+    const { size, weight = "normal", style = "normal", family } = font;
+    const fontString = `${style} ${weight} ${size ? `${size}px` : ""} ${
+      family ? `"${family}"` : ""
```

Comment:
Found a problem: generic names like `sans-serif` shouldn't be quoted. I think whether to quote or not should be up to the game code, not here.

Reply:
That's now resolved, thanks!